Dockerfile

```
@@ -0,0 +1,36 @@
+ARG NODE_IMAGE_VERSION
+
+# Create base image
+FROM node:${NODE_IMAGE_VERSION} as base
+RUN apk add --no-cache tini python make g++
+
+# Install atlas engine
+FROM base as atlas_engine
```

Bitte den vollen Namen des Paketes verwenden. WIr haben in der AtlasEngine ja mehr als eine Anwendung, die man dockerizen kann.

```suggestion
FROM base as atlas_engine_fullstack_server
```